x/oracle/keeper/oracle_test.go

```diff
@@ -264,7 +264,7 @@ func (suite *oracleTestSuite) TestApproveOracleRegistrationSuccess() {
 	encryptedOraclePrivKey, err := btcec.Encrypt(suite.nodePubKey, suite.oraclePrivKey.Serialize())
 	suite.Require().NoError(err)
 
-	approveOracleRegistration := &types.ApproveOracleRegistration{
+	approveOracleRegistration := &types.ApprovalSharingOracleKey{
```

Comment:
I think that variable name also changes to `approvalSharingOracleKey`.

Reply:
thank you I'll change it!